### PR TITLE
IBX-5715: Added validate if anchor value is unique and valid

### PR DESCRIPTION
**Pull request by mateuszdebinski**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [IBX-5715](https://issues.ibexa.co/browse/IBX-5715)
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | 4.5
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

I had no other idea to verify if an element is unique, the previous solution doesn't work anymore. I'd love to hear other ideas for checking the uniqueness of the ID

**TODO**:
- [x] Implement feature / fix a bug.
- [ ] Implement tests.
- [ ] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

**Review by dew326 — APPROVED** [review]

**Review by lucasOsti — APPROVED** [review]

**Review by Gengar-i — APPROVED** [review]

File: src/bundle/Resources/public/js/CKEditor/anchor/anchor-ui.js [hunk 1]
```diff
@@ -87,6 +98,29 @@ class IbexaAnchorUI extends Plugin {
...
+        let countElementsWithSameId = 0;
+
+        Array.from(richtexts).every((richtext) => {
+            if (countElementsWithSameId >= 2) {
+                return false;
+            }
+
+            countElementsWithSameId += richtext.querySelectorAll(`[id="${value}"]`).length;
+
+            return true;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/CKEditor/anchor/anchor-ui.js:114]:
```suggestion
            countElementsWithSameId += richtext.querySelectorAll(`[id="${value}"]`).length;

            return countElementsWithSameId > 1;
```

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Comment by micszo** [comment]
@mateuszdebinski what's the decision regarding CI state?

**Comment by mateuszdebinski** [comment]
@micszo I did a rebase, and now it's ok

**Review by micszo — COMMENTED** [review]
Following error occurs when clicking Save:

Screenshot 2023-07-04 at 13 34 57

**Review by micszo — APPROVED** [review]
QA Approved on Ibexa Experience 4.5.1-dev.

**Pull request merged**